aspnetcore/blazor/javascript-interoperability/import-export-interop.md

```diff
@@ -259,6 +224,9 @@ export async function setMessage() {
 }
 ```
 
+> [!NOTE]
+> Calling `globalThis.getDotnetRuntime(0)` to obtain the exports can occur in a [JavaScript initializer](xref:blazor/js-interop/index#javascript-initializers) for availability across the app.
```

Comment:
This kind of makes it sound like `getDotnetRuntime(0)` is obtaining the exports, but that's what `getAssemblyExports` is doing. Maybe changing it to...
```suggestion
> Calling `getAssemblyExports` to obtain the exports can occur in a [JavaScript initializer](xref:blazor/js-interop/index#javascript-initializers) for availability across the app.
```
...is sufficient?